mobile/components/CustomButton.tsx

```diff
@@ -12,20 +12,28 @@ import {StyleSheet, Text, Pressable, Alert} from 'react-native';
   Below is a premade button without any props -- check out its syntax!
   Use the StyleSheet below to customize the button with classes.
 */
-const CustomButton = () => {
-return(
-  <Pressable onPress={() => Alert.alert('Simple Button pressed!')} style={styles.button1}> 
-    <Text>I'm a button!</Text>
-  </Pressable>
-)};
+
+type ButtonProps = {
+  text: string,
+  handleClick: () => {},
```

Comment:
Try to make the typing a little more clear
```suggestion
  handleClick: () => object,
```